shell/gpu/gpu_surface_metal_impeller.mm

```diff
@@ -192,7 +192,7 @@
   if (!disable_partial_repaint_) {
     // Provide accumulated damage to rasterizer (area in current framebuffer that lags behind
     // front buffer)
-    uintptr_t texture = reinterpret_cast<uintptr_t>(drawable.texture);
+    void* texture = (__bridge void*)drawable.texture;
```

Comment:
ARC implicit type conversion really doesn't want to allow `(__bridge uintptr_t)`, maybe it's... a bridge too far. Yeah I could reinterpret_cast the void* to a uintptr_t now that the pointer is bridged, but this seems fine.

Reply:
Adjusting the types so that we no longer need the `reinterpret_cast`s seems like pure win to me. The fact that it bypassed the compiler warning about needing a bridge is more evidence of how dangerous it is.